source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/APIControllers/UsersTest.cs

```diff
@@ -118,5 +120,26 @@ public async Task Patch_SomeUser_Different_Organization_NotFound()
 
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
+
+        /// <summary>
+        /// Getting the current user (edit the user profile) when a different
+        /// organization is the current organization (e.g. viewing users from
+        /// a different organization) should always be allowed and return the
+        /// current user.
+        /// </summary>
+        /// <returns>The current user different organization found.</returns>
+        [Fact]
+        public async Task Get_CurrentUser_Different_Organization_Found()
+        {
+            var user = await GetCurrentUser();
+
+            NeedsTestData<AppDbContext, Organization>(new List<Organization>
+            {
+                new Organization { Name = "Foo"}
+            });
+
+            var response = await Get($"/api/users/{user.Id}", "Foo");
```

Comment:
I believe the org second arg should be an org id. Even though

Reply:
Ah yes...